insight/tasks/weekly_llm_analyzer.py

```
@@ -0,0 +1,134 @@
+    "너는 세계 최고의 50년차 트랜드 분석 전문가야. 기술 블로그 글 데이터를 기반으로 주간 뉴스레터를 작성해야 해.\n"
+    "내가 제공하는 데이터만 활용해서 해당 내용의 트랜드를 파악하고 요약해야 해. 필요하면 관련된 외부 검색도 해줘."
+)
+
+WEEKLY_TREND_PROM = """
+<목표>
+- 블로그 글 데이터의 트렌드 분석
+- 분석 세부 내용은 "전체 인기글, 기술 키워드, 제목 트렌드, 글의 상세 내용의 요약 및 트랜드" 파악
+
+<작성 순서>
+1. 🔥 주간 트렌딩 글 요약
+	- 아래에 제공한 모든 트렌딩 글 핵심 내용 요약
```

Flake8 (7.2.0)

[error] indentation contains mixed spaces and tabs

(E101)